packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -69,6 +71,38 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             Lib_ExecutionManagerWrapper.ovmCHAINID()
         );
 
+        // Compute the intrinsic gas
```

Comment:
I find this incredibly ugly. Perhaps we use a library for computing the intrinsic gas?